<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QMainWindow" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToolBar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>539</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveauTableauAleatoire"/>
    <addaction name="actionOuvrirTableau"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionExporter"/>
    <addaction name="actionImprimer"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRestaurer"/>
    <addaction name="separator"/>
    <addaction name="actionToutSelectionner"/>
    <addaction name="separator"/>
    <addaction name="actionCopier"/>
    <addaction name="separator"/>
    <addaction name="actionFusionner"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionZoomPlus"/>
    <addaction name="actionZoomMoins"/>
    <addaction name="actionPleinEcran"/>
    <addaction name="separator"/>
    <addaction name="actionBarreDEtat"/>
    <addaction name="actionBarreDOutils"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionModeSombre"/>
    <addaction name="separator"/>
    <addaction name="actionLangues"/>
    <addaction name="separator"/>
    <addaction name="actionChoisirCouleurs"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAnnulerTool"/>
   <addaction name="actionRestaurerTool"/>
   <addaction name="actionZoomPlusTool"/>
   <addaction name="actionZoomMoinsTool"/>
   <addaction name="actionFusionTool"/>
   <addaction name="actionOuvrirTool"/>
   <addaction name="actionChoisirCouleurTool"/>
  </widget>
  <widget class="QStatusBar" name="barreDeStatut"/>
  <action name="actionNouveauTableauAleatoire">
   <property name="text">
    <string>Nouveau tableau aleatoire</string>
   </property>
  </action>
  <action name="actionOuvrirTableau">
   <property name="text">
    <string>Ouvrir tableau</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionRestaurer">
   <property name="text">
    <string>Restaurer</string>
   </property>
  </action>
  <action name="actionZoomPlus">
   <property name="text">
    <string>Zoom avant</string>
   </property>
  </action>
  <action name="actionZoomMoins">
   <property name="text">
    <string>Zoom arriere</string>
   </property>
  </action>
  <action name="actionPleinEcran">
   <property name="text">
    <string>Pleine ecran</string>
   </property>
  </action>
  <action name="actionBarreDEtat">
   <property name="text">
    <string>Barre d'etats</string>
   </property>
  </action>
  <action name="actionBarreDOutils">
   <property name="text">
    <string>Barre d'outils</string>
   </property>
  </action>
  <action name="actionModeSombre">
   <property name="text">
    <string>Mode sombre</string>
   </property>
  </action>
  <action name="actionLangues">
   <property name="text">
    <string>Langues</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionToutSelectionner">
   <property name="text">
    <string>Tout selectionner</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionAnnulerTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/ninja.png</normaloff>../../../../../../Images chelou/ninja.png</iconset>
   </property>
   <property name="text">
    <string>Annuler Tool</string>
   </property>
   <property name="toolTip">
    <string>Annuler</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRestaurerTool">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/jesus.jpeg</normaloff>../../../../../../Images chelou/jesus.jpeg</iconset>
   </property>
   <property name="text">
    <string>Restaurer Tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomPlusTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/Layton.jpeg</normaloff>../../../../../../Images chelou/Layton.jpeg</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool Plus</string>
   </property>
  </action>
  <action name="actionZoomMoinsTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/ZoomOut.png</normaloff>../../../../../../Images chelou/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool Moins</string>
   </property>
  </action>
  <action name="actionFusionTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/fusion.jpeg</normaloff>../../../../../../Images chelou/fusion.jpeg</iconset>
   </property>
   <property name="text">
    <string>Fusion Tool</string>
   </property>
  </action>
  <action name="actionOuvrirTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/fbi.jpeg</normaloff>../../../../../../Images chelou/fbi.jpeg</iconset>
   </property>
   <property name="text">
    <string>Ouvrir Tool</string>
   </property>
  </action>
  <action name="actionChoisirCouleurTool">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Images chelou/lgbtflag.jpeg</normaloff>../../../../../../Images chelou/lgbtflag.jpeg</iconset>
   </property>
   <property name="text">
    <string>Choisir Couleur Tool</string>
   </property>
  </action>
  <action name="actionChoisirCouleurs">
   <property name="text">
    <string>Choisir couleurs</string>
   </property>
  </action>
  <action name="actionFusionner">
   <property name="text">
    <string>Fusionner</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
